.github/workflows/automatic-updates.yml

```
@@ -0,0 +1,52 @@
+name: Automatically update Docker image versions
+
+on:
+  schedule:
+    - cron: "*/15 * * * *"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    # if the repo owner isn't nodejs and the branch isn't update-branch
```

```suggestion
    # if the repo owner isn't nodejs and the branch isn't update-branch
```

comment just explains the code, but also there's no branch check?